nilearn/decoding/decoder.py

```diff
@@ -619,6 +619,11 @@ def _set_classes(self, y):
         raise NotImplementedError()
 
     def _n_problems(self) -> int:
+        """Define the number problems to solve.
+
+        In case of classification this
+        number corresponds to the number of binary problems to solve.
+        """
         if len(self._get_classes()) > 2:
```

Comment:
Should this be `>= 2` ? If not, in the below replacement, self._n_problems will never return 2.

Reply:
indeed